Core/include/Acts/SpacePointFormation/SpacePointBuilderConfig.h

```diff
@@ -0,0 +1,45 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2021 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "Acts/Definitions/Units.hpp"
+#include "Acts/Geometry/TrackingGeometry.hpp"
+
+namespace Acts {
+struct SingleHitSpacePointBuilderConfig{
+   // Tracking geometry
+   std::shared_ptr<const Acts::TrackingGeometry> trackingGeometry; 
+
+   SingleHitSpacePointBuilderConfig()=default;
+};
+
+
+ struct DoubleHitSpacePointBuilderConfig {
+  // Tracking geometry
+  std::shared_ptr<const Acts::TrackingGeometry> trackingGeometry; 
+  /// Accepted squared difference in theta for two clusters
+  double diffTheta2 = 1.;
+  /// Accepted squared difference in phi for two clusters
+  double diffPhi2 = 1.;
+  /// Accepted distance between two clusters
+  double diffDist = 100. * UnitConstants::mm;
+  /// Allowed increase of strip length
+  double stripLengthTolerance = 0.01;
+  /// Allowed increase of strip length wrt gaps between strips
+  double stripLengthGapTolerance = 0.01;
+  /// Assumed position of the vertex
+  Vector3 vertex = {0., 0., 0.};
+  /// Perform the perpendicular projection for space point finding
+  bool usePerpProj = false;
+
+  DoubleHitSpacePointBuilderConfig()=default;
+};
```

Comment:
Hi @toyamaza, why are these here but you also have a `Config` struct inside the SingleHitSpacePointBuilder class definition?

What about having a struct `SingleHitSpacePointBuilderConfig` with only a `std::shared_ptr<const Acts::TrackingGeometry> trackingGeometry; ` and then have `DoubleHitSpacePointBuilderConfig` inherit from it adding the remaining properties?